src/services/sign-interpretation.ts

```diff
@@ -43,7 +43,15 @@ export default class SignInterpretationService {
             commentary: attribute.commentary?.commentary
         };
 
-        const siDto = await CommHelper.post<SignInterpretationDTO>(url, dto);
-        return siDto;
+        const response = await CommHelper.post<SignInterpretationDTO>(url, dto);
+        return response.data;
+    }
+
+    public async updateCommentary(edition: EditionInfo, signInterpretation: SignInterpretation) {
+        const url = ApiRoutes.signInterpretationCommentaryUrl(edition.id, signInterpretation.id);
+        const dto: CommentaryCreateDTO = { commentary: signInterpretation.commentary || undefined };
+
+        const response = await CommHelper.put<SignInterpretationDTO>(url, dto);
+        return response.data;
```

Comment:
I really like how this function looks.  It is clear and all of the complexity is abstracted away.  The CommHelper system really looks good.